projects/demo-playwright/tests/kit/dropdown-hover/dropdown-hover.pw.spec.ts

```diff
@@ -1,84 +0,0 @@
-import {DemoRoute} from '@demo/routes';
-import {
-    TuiDocumentationPagePO,
-    tuiGoto,
-    TuiMobileDropdownPO,
-} from '@demo-playwright/utils';
-import {expect, type Locator, test} from '@playwright/test';
-
-import {TUI_PLAYWRIGHT_MOBILE} from '../../../playwright.options';
-
-const {describe, beforeEach} = test;
-
-test.describe('DropdownHover', () => {
```

Comment:
Why this test was deleted ?

It checks that previously existed bug is not reproduced again:
* https://github.com/taiga-family/taiga-ui/issues/9589